integration-tests/scripts/exporter-node.sh

```diff
@@ -34,9 +39,13 @@ setup(){
 
   sed -i "s@DestinationShardAsObserver =.*@DestinationShardAsObserver = \"$1\"@" $DEVEL/config/prefs.toml
   sed -i 's/FullArchive =.*/FullArchive = true/' $DEVEL/config/prefs.toml
-  sed -i "s@reader-node-path:.*@reader-node-path: \"$DEVEL/node\"@" $DEVEL/standard.yaml
 
-  ./start.sh
+  sed -i '/HostDriverConfig\]/!b;n;n;c\    Enabled = true' $DEVEL/config/external.toml
+  sed -i "s@Mode =.*@Mode = \"$OBSERVER_MODE\"@" $DEVEL/config/external.toml
+  sed -i 's/MarshallerType =.*/MarshallerType = "gogo protobuf"/' $DEVEL/config/external.toml
```

Comment:
maybe you can add also a variable for the marshaller type

Reply:
No. Should be hard codded. It is mandatory that the marshaller is proto